doc/how-to/remove_machine.md

```diff
@@ -10,8 +10,29 @@ Before removing the machine, ensure that there are no LXD instances or storage v
 See {ref}`how to remove instances <lxd:instances-manage-delete>` in the LXD documentation.
 See {doc}`how to remove OSDs <microceph:how-to/remove-disk>` in the MicroCeph documentation.
 
+```{note}
+If local storage was created, MicroCloud will have also added some default storage volumes that will need to be cleaned up:
+
+    lxc config unset storage.images_volume --target <name>
+    lxc config unset storage.backups_volume --target <name>
+    lxc storage volume delete local images --target <name>
+    lxc storage volume delete local backups --target <name>
+
+Any additional storage volumes belonging to this machine must also be deleted before removal without the `--force` flag.
```

Comment:
Can we run those cleanup steps against the LXD API if the user provides `--force`? 
Having to provide `--force` but still requiring to delete resources upfront seems a bit too much.

Reply:
`--force` is basically "skip cleanup". You`d use `--force` if the node is not available for cleanup in the first place, like if the node is unreachable. To get below 3 nodes, MicroCeph requires us to pass `--force`, and we cant manage underlying Ceph/OVN cleanup, so regardless of any cleanup we do on LXD, the node would never be sufficiently "cleaned" anyway. 

Also I'm generally hesitant to delete user-created volumes & instances silently.

Reply:
Also we try to be proactive, so if you run `microcloud remove` without `--force`, then everything except MicroCeph and MicroCloud will be removed cleanly. Then you would need to rerun `microcloud remove --force` for the same node to remove the last 2 services.

Reply:
thanks!